extern/algebra-plugins/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building Algebra Plugins as part of the TRACCC project" )
+
+# Declare where to get Algebra Plugins from.
+set( TRACCC_ALGEBRA_PLUGINS_SOURCE
+   "URL;https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.3.0.tar.gz;URL_MD5;7ffcfe299ebf654074b196e949afb010"
```

Comment:
I think we might eventually want to store these URLs and hashes somewhere outside of the CMake configuration as a single source of truth, but this is fine for now.

Reply:
I guess so. But in my mind that would just unnecessarily complicate the CMake code.

These should be set up as cache variables in order to allow us to override them if/when necessary. So if we had a text file called `externals.txt` that we would read all those values out of, we would have to write some non-trivial CMake code that did this during configuration, and then still set up a cache variable with the value coming from that text file. Since I'd still want the value to be overridable.

Reply:
Sure, let's skip this for now.